palm/plugins/dbt/commands/cmd_dbt.py

```diff
@@ -0,0 +1,92 @@
+from email.policy import default
+from unittest.result import failfast
+import click
+from typing import Optional, Tuple
+from palm.plugins.dbt.dbt_palm_utils import dbt_env_vars
+
+
+@click.command("dbt")
+@click.option("--select", '-s', help="Specify the nodes to include.")
+@click.option("--exclude", '-e', help="Specify the nodes to exclude.")
```

Comment:
I think this is needed to accept more than one model and to make the join you added below work
```suggestion
@click.option("--select", '-s', multiple=True, help="Specify the nodes to include.")
@click.option("--exclude", '-e', multiple=True, help="Specify the nodes to exclude.")
```

Reply:
yeah, you right!